wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsplugins/model/PluginMeta.java

```diff
@@ -153,6 +154,9 @@ public PluginMeta tags(List<String> tags) {
   }
 
   public List<String> getTags() {
+    if (tags == null) {
+      return new ArrayList<>();
```

Comment:
It should be assigned

Reply:
fixed, also added initialization in the field itself